kotlin-bundled-compiler/get_bundled.xml

```diff
@@ -196,6 +197,17 @@
     	           </patternset>
     	           <mapper type="flatten"/>
     	       </unzip>
+
+        <unzip src="${download.dir}/${compiler-artifact-file}" dest="${target.dir}">
```

Comment:
Note that this might be very dangerous as we use proGuard on our jars and many declarations can be lost from this version of kotlinx.coroutines. I think it can live for now, but in gradle model, this library should be downloaded from `mavenCentral`